css/styles.css

```
@@ -7880,133 +7873,578 @@ a.about-badge:hover {
+}
+.dm-series-chip[aria-pressed="false"] {
+  opacity: 0.45;
+}
+.dm-series-chip:disabled {
+  opacity: 0.3;
+  cursor: not-allowed;
+}
+.dm-series-swatch {
+  width: 14px;
+  height: 0;
+  border-top: 3px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)